### ISSUE TITLE
Nagios should cache RSS feeds on the home page

### ISSUE THREAD

**Issue by tmcnag** [NagiosEnterprises/nagioscore#184]
> This issue has been automatically migrated from [http://tracker.nagios.org/view.php?id=0000235](http://tracker.nagios.org/view.php?id=0000235)  - Please follow that link for details.
> Originally posted on 2011-08-04 by agriffin

Currently with the new home page on the Nagios Core web interface, the new RSS feeds are fetched with every page load. It should be cached, and probably somehow handled better even with a cache (AJAX or something).

In some cases this may be preventing the page from loading at all. See http://support.nagios.com/forum/viewtopic.php?f=7&t=2975

**Comment by jfrickson** [comment]
On 2011-08-06, dnsmichi responded:

> it should be possible to completely disable the home calls when opening the main page. make it a cgi.cfg option or similar.

**Comment by ericloyd** [comment]
I disagree that it should be cached.  It's up to your network to provide caching if that's what you want.  Reverse transparent proxy would handle that easily enough.

**Comment by tmcnag** [comment]
@ericloyd Agreed - For clarity's sake, my original comment in this was an automated comment from the migration.  I personally don't think the caching should be done in Core.

**Comment by ericloyd** [comment]
TL;DR.  :-)

**Comment by jfrickson** [comment]
Closing. Not core's task, per discussion above.
